newIDE/app/src/Profile/CreateAccountDialog.js

```diff
@@ -149,54 +159,86 @@ const CreateAccountDialog = ({
       maxWidth="sm"
       open
     >
-      <ColumnStackLayout noMargin>
+      <ColumnStackLayout
+        noMargin
+        expand
+        justifyContent="center"
+        alignItems="center"
+      >
+        <GDevelopGLogo fontSize="large" />
+        <Text size="title">
+          <Trans>Sign up for free!</Trans>
+        </Text>
+        <Column noMargin alignItems="center">
+          <Text size="body2" noMargin>
+            <Trans>Welcome to GDevelop!</Trans>
+          </Text>
+          <LineStackLayout noMargin>
+            <Text size="body2" noMargin>
+              <Trans>Already a member?</Trans>
+            </Text>
+            <Link
+              href="#"
+              onClick={onGoToLogin}
+              disabled={createAccountInProgress}
+            >
+              <Text size="body2" noMargin color="inherit">
+                <Trans>Log in to your account</Trans>
+              </Text>
+            </Link>
+          </LineStackLayout>
+        </Column>
+        <div style={styles.formContainer}>
+          <ColumnStackLayout noMargin>
+            <UsernameField
+              value={username}
+              onChange={(e, value) => {
+                setUsername(value);
+              }}
+              allowEmpty
+              onAvailabilityChecked={setUsernameAvailability}
+              onAvailabilityCheckLoading={setIsValidatingUsername}
+              isValidatingUsername={isValidatingUsername}
+              disabled={createAccountInProgress}
+            />
+            <TextField
+              value={email}
+              floatingLabelText={<Trans>Email</Trans>}
+              errorText={getEmailErrorText(error)}
+              fullWidth
+              required
+              onChange={(e, value) => {
+                setEmail(value);
+              }}
+              disabled={createAccountInProgress}
+            />
+            <TextField
+              value={password}
+              floatingLabelText={<Trans>Password</Trans>}
+              errorText={getPasswordErrorText(error)}
+              type="password"
+              fullWidth
+              required
+              onChange={(e, value) => {
+                setPassword(value);
+              }}
+              disabled={createAccountInProgress}
+            />
+            <Checkbox
+              label={<Trans>I want to receive the GDevelop Newsletter</Trans>}
+              checked={getNewsletterEmail}
+              onCheck={(e, value) => {
+                setGetNewsletterEmail(value);
+              }}
+              disabled={createAccountInProgress}
+            />
+          </ColumnStackLayout>
+        </div>
         <BackgroundText>
```

Comment:
I wonder if this should be a Text with color secondary and a link rather than a background text with markdown

Reply:
As this is more of a "keep in mind these are the T&C's, you can click if you want" I like the fact that it's a greyer text, not appearing as a main action.
I think it's fine